site/src/docs/components/highlight/accessibility.mdx

```diff
@@ -0,0 +1,17 @@
+---
+slug: '/components/highlight/accessibility'
+title: 'Highlight - Accessibility'
+---
+
+import ExternalLink from '../../../components/ExternalLink';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Accessibility
+- Both the Quote and Highlight elements are non-interactable and therefore do not have any special keyboard functionality.
+
+### Pay attention to
+- Both the Quote and Highlight components have three size variants; Large (l), Medium (m) and Small (s)
+- Sizing variants differ in typography size and inner spacing.
+- Both the Higlight and Quote adds vertical margin around the actual content
```

Comment:
Hmm, I don't know if these are accessibility related notes? I think it would be ok to just remove this "Pay attention to" section.